docs/installation/packages.rst

```diff
@@ -30,12 +30,15 @@ The following list may be incomplete, feel free to submit a PR with an update:
     * - Arch Linux
       - ``yay install zrepl``
       - Available on `AUR <https://aur.archlinux.org/packages/zrepl>`_
-    * - Fedora
+    * - Fedora + CentOS 8
       - ``dnf install zrepl``
-      - Available on `COPR <https://copr.fedorainfracloud.org/coprs/poettlerric/zrepl/>`_
-    * - CentOS/RHEL
+      - Available on `Fedora Copr <https://copr.fedorainfracloud.org/coprs/awehrfritz/zrepl/>`_
+    * - CentOS 7
       - ``yum install zrepl``
-      - Available on `COPR <https://copr.fedorainfracloud.org/coprs/poettlerric/zrepl/>`_
+      - Available on `Fedora Copr <https://copr.fedorainfracloud.org/coprs/awehrfritz/zrepl/>`_
+    * - openSUSE
+      - ``zypper install zrepl``
+      - Available on `Fedora Copr <https://copr.fedorainfracloud.org/coprs/awehrfritz/zrepl/>`_
```

Comment:
This is good for now (i.e. version 0.3.0) as my Copr project is still live and in fact uses the original spec file from a separate git repo, but eventually this should be updated to point to the new rpm repo (with instructions on how to add that new repo).

Reply:
My plan is to offer the RPM repo as of 0.3.1 which I plan to release in a few weeks at most.